index.js

```diff
@@ -17,7 +17,7 @@ class Client {
    * @param {boolean} [options.raw=false] Makes it so that we return the raw string value. Default is false.
    */
   async get(key, options) {
-    return await fetch(this.key + "/" + key)
+    return await fetch(this.key + '/' + key)
```

Comment:
we need to give this the same treatment, no?

Reply:
Good catch, will fix